src/ScheduledExecutorService.md

```diff
@@ -338,6 +342,7 @@ Following are the descriptions of each configuration element and attribute:
 
 * `name`: Name of the scheduled executor.
 * `pool-size`: Number of executor threads per member for the executor.
+* `capacity`: Max number of tasks that a Scheduler can have per partition. Attempt to schedule more, will result in RejectedExecutionException. To free-up capacity, tasks should get disposed by the user.
 * `durability`: Durability of the executor.
```

Comment:
* `capacity`: Maximum number of tasks that a Scheduler can have per partition. Attempt to schedule more, will result in `RejectedExecutionException`. The tasks should get disposed by the user to free-up the capacity.